js/stanford_paragraph_types.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file
+ * Applies some functionality on the paragraph types.
+ */
+
+(function ($) {
+  Drupal.behaviors.stanfordParagraphTypes = {
+    attach: function (context, settings) {
+      $('.field-name-field-p-hero-image').once(function () {
+        if ($(this).siblings('.field-name-field-p-hero-video').length) {
+          var play = $('<a>', {
+            class: 'play-video',
+            href: '#',
+            onclick: 'return false;',
```

Comment:
This should be added as an event handler with the event.preventDefault() suppression on the click event.

`eg: $("<a>", {...}).click(function(e){e.preventDefault();});`